README.md

```diff
@@ -643,6 +643,7 @@ Table of Contents
   * [K8Spin](https://k8spin.cloud) - Managed Kubernetes namespace service designed for developers. Free developer accounts come with with 100 mCore of CPU, 128MB of RAM.
   * [ZARVIS](https://zarvis.ai) - Free managed Kubernetes namespace for open source Github project. Free 1GB memory and 1 vCPU.
   * [mendix.com](https://www.mendix.com/) — Rapid Application Development for Enterprises, unlimited number of free sandbox environments supporting 10 users, 100 MB of files and 100 MB database storage each
+  * [m3o.com](https://m3o.com) - A cloud platform for API services development. M3O is a fully managed Micro as a Service offering focusing on Go microservices development in the Cloud. Free tier provides enough to run 5 services and collaborate with others.
```

Comment:
Is the signup processes documented there all thats needed for the free tier or MUST you first do `To get access please talk to #m3o-platform on slack.`?

Reply:
Signup process is documented on the landing page as `micro signup`. Its still invite only by way of talking to us.

Reply:
OK, unfortunately in the past we've not added invite only/human based signup free services here, so afraid I'll have to do the same here.  Has to be low touch

Reply:
OK. Please leave this here for now. We'll open up the dev tier without invite requirements on October 20th. So I'll ping again when its done.

Reply:
ok sweet

Reply:
Hey @ripienaar. The Dev tier went live today. Developers will see instructions on the website and related docs for using the free environment by simply switching using `micro env set dev`. Can you please merge this PR at your earliest convenience. Thanks.

Reply:
https://m3o.dev/concepts/platform#dev-environment there? still asks for slack or do you mean you're in process of rolling that out?

Reply:
Fixed. Its just a wording issue. The sign-up is totally open.

Reply:
good stuff, thanks for sticking with it :)

Reply:
Thank you for merging!